torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -398,6 +416,82 @@ def _annotate_for_static_quantization_config(
 
         return model
 
+    def _annotate_qat_conv2d_fusion_pattern(
+        self, model: torch.fx.GraphModule, config: QuantizationConfig
+    ):
+        # Annotate QAT Specific patterns
+        self._annotate_qat_conv2d_bn_unary(model, config)
```

Comment:
do you need to add something to prepare_pt2e code to support this pattern? https://github.com/pytorch/pytorch/blob/main/torch/ao/quantization/quantize_pt2e.py#L178

we are also thinking about moving qat specific code to quantizer as well actually, we'll discuss a bit internally and get your feedback

Reply:
Yean, for now we only have relu as unary post op which I think is already supported in https://github.com/pytorch/pytorch/blob/main/torch/ao/quantization/quantize_pt2e.py#L178.